### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'duration')

### ISSUE THREAD

**Issue by resteqs** [Werter12/material-ui-audio-player#487]
@Werter12 what do i need to do?
TypeError: Cannot read properties of null (reading 'duration')
(anonymous function)
C:/Users/src/components/state/actions.ts:61
  58 |         dispatch({ type: PLAYER_VOLUME_STATUS_UNMUTE });
  59 |     };
  60 | }
> 61 | function changeAudioVolume(dispatch, player) {
  62 |     return function (value) {
  63 |         if (player.current) {
  64 |             player.current.volume = value > 0 ? value / 100 : 0;

**Comment by resteqs** [comment]
Hello ? ;d

**Comment by lefnire** [comment]
In my case this is an unmount issue (navigating away from the page with the player). Still investigating

**Comment by Wideyedwonderer** [comment]
I have the same issue. It seems to have appeared once I migrated to the newer version of material UI(currently experimenting with having both versions, which might not be the best of ideas). It also seems to appear only on iPhone mobile version.

**Comment by nb312** [comment]
i have the same problem:TypeError: Cannot read properties of null (reading 'duration')

**Comment by 9267aakashsharma** [comment]
Yes I encountered the same issue as well. It is happening on un-mount.

**Comment by ghadeeraqraa1992** [comment]
The same issue here, any fix ?
